Source/AliveLib/Abe.hpp

```diff
@@ -176,6 +176,51 @@ enum class AbeSay : __int16
     eSympathy_28 = 28,
 };
 
+const enum class sAbeSFXList_555250_Tag : unsigned __int8
```

Comment:
const makes no sense here and why _Tag?

Reply:
Because those are labels for sAbeSFXList_555250[] sounds